tools/sample_generator/templates/default/WinUI.xaml.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright sample_year Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks;
+using Esri.ArcGISRuntime.Tasks.Offline;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.ArcGISServices;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Linq;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Input;
+
+namespace ArcGISRuntime.WinUI.Samples.sample_name
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "friendly_name",
+        "sample_category",
+        "sample_description",
+        "")]
+    [offline_data_attr]
+    public partial class sample_name
+    {
+        public sample_name()
+        {
+            InitializeComponent();
+            Initialize();
+        }
+
+        private void Initialize()
```

Comment:
Should this be async Task?

Reply:
There aren't always async calls in initialize, so this should be changed when async calls are used.

Reply:
@ZackAllen That's a fair point. Seems like more often than not though, we're creating maps/scenes/layers/symbols and they're all async operations we should be awaiting (or handling load status changed).